addons/l10n_jp/data/account_tax_template_data.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="tax_in_e" model="account.tax.template">
+    <record id="l10n_jp_tax_in_e" model="account.tax.template">
```

Comment:
I think we can do a better job at making the 'in'/'out' and 'e'/'i' of l10n_jp_tax_in_e more explicit? maybe put sale/purchase  and exc / incl

+ why not also adding the amount in those one _8 ?